src/mono/mono/profiler/browser.c

```diff
@@ -93,11 +292,19 @@ mono_profiler_init_browser (const char *desc)
 		return;
 	}
 
-#ifdef HOST_WASM
+#ifdef HOST_BROWSER
+	profiler_stack = NULL;
```

Comment:
We should make sure these are initialized even if aot_compiling so they're not garbage, since the interpreter can run in AOT mode

Reply:
only `method_filter` is executed in AOT and those variables don't even exist there.